corehq/apps/cloudcare/views.py

```diff
@@ -245,14 +246,16 @@ def form_context(request, domain, app_id, module_id, form_id):
 
 
 def get_cases_vary_on(request, domain):
+    request_params = request.GET if request.method == 'GET' else request.POST
```

Comment:
pretty sure these are all GET requests, but this is probably safer. might be interesting to add a soft assert